content/applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst

```diff
@@ -0,0 +1,153 @@
+======================
+Two-step manufacturing
+======================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Odoo *Manufacturing* allows users to manufacture products using one, two, or three steps. When using
+two-step manufacturing, Odoo creates a manufacturing order and a pick components transfer, but does
+not generate a transfer for the movement of finished products into stock. Inventory counts still
+update based on the number of products manufactured, but the act of transferring them to and from
+inventory is not tracked.
+
+.. tip::
+   The number of steps used in manufacturing is set at the warehouse level, allowing for each
+   warehouse to use a different number of steps. To change the number of steps used for a specific
+   warehouse, begin by navigating to :menuselection:`Inventory --> Configuration --> Warehouses`,
+   then select a warehouse from the :guilabel:`Warehouses` screen.
+
+   On the :guilabel:`Warehouse Configuration` tab, find the :guilabel:`Manufacture` radio input field
+   and select one of the three options: :guilabel:`Manufacture (1 step)`, :guilabel:`Pick components
+   and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then store
+   products (3 steps)`.
```

Comment:
The "d" in "field" in the first line went beyond the 100 character max, so I reformatted the paragraph for you :smiley: 

```suggestion
   On the :guilabel:`Warehouse Configuration` tab, find the :guilabel:`Manufacture` radio input
   field and select one of the three options: :guilabel:`Manufacture (1 step)`, :guilabel:`Pick
   components and then manufacture (2 steps)`, or :guilabel:`Pick components, manufacture and then
   store products (3 steps)`.
```